STM32/LightController/Core/Src/main.c

```diff
@@ -214,15 +208,14 @@ static void MX_TIM2_Init(void)
 
   TIM_ClockConfigTypeDef sClockSourceConfig = {0};
   TIM_MasterConfigTypeDef sMasterConfig = {0};
-  TIM_OC_InitTypeDef sConfigOC = {0};
 
   /* USER CODE BEGIN TIM2_Init 1 */
 
   /* USER CODE END TIM2_Init 1 */
   htim2.Instance = TIM2;
-  htim2.Init.Prescaler = 63;
+  htim2.Init.Prescaler = 0;
   htim2.Init.CounterMode = TIM_COUNTERMODE_UP;
-  htim2.Init.Period = 255;
+  htim2.Init.Period = 799;

```

Comment:
Not important, but was considering it might be helpful, particularly for reviewers or people taking over to set the prescaler to the next smallest sensible interval.
E.g. for 20 kHz timer down from 16 MHz, the prescaler would be 15 (so 1 period = 1 us) and the time would be 50.
I think it would be more understandable for reviewing, but also quicker for iterating in development.

Anyway, not crucial, just an idea I had